common/lvm/util.go

```diff
@@ -132,14 +133,24 @@ func (lvmNodesDevicePvVgConfig *LvmNodesDevicePvVgConfig) ConfigureLvmNodesWithD
 
 func SetupLvmNodes(vgName string, size int64) (LvmNodesDevicePvVgConfig, error) {
 	var lvmNodeConfig LvmNodesDevicePvVgConfig
-	loopDevice := e2e_agent.LoopDevice{
-		Size:   size,
-		ImgDir: "/tmp",
-	}
 	workerNodes, err := ListLvmNode(common.NSOpenEBS())
 	if err != nil {
 		return lvmNodeConfig, fmt.Errorf("failed to list lvm worker nodes, error: %v", err)
 	}
+	if len(workerNodes) == 0 {
+		return lvmNodeConfig, fmt.Errorf("lvm worker nodes not found")
+	}
+	var imgDir string
+	if strings.Contains(workerNodes[0], "kind-") {
```

Comment:
Please add comments describing what is being detected here and assumptions/expectations of the cluster/node setup

Reply:
Added comments https://github.com/openebs/openebs-e2e/pull/65/commits/560dcf6648ec842d8c1f104a8d58f9dc1ed8f129